modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,198 @@ The properties for broker-wide throughput quota balancing are configured at the
+Alternatively, you can view or configure throughput quotas for a group of clients based on a match on client ID prefix. The following example sets the `consumer_byte_rate` quota to client IDs prefixed with `consumer-`:
+
+[,bash]
+----
+rpk cluster quotas alter --add consumer_byte_rate=200000 --name client-id-prefix=consumer-
+----
+
+NOTE: A client group specified with `client-id-prefix` is not the equivalent of a Kafka consumer group. It is used only to match requests based on the `client_id` prefix. The `client_id` field is typically a configurable parameter when you create a client with Kafka libraries.
+
+=== Default client throughput limit
+
+You can apply default throughput limits to clients. Redpanda applies the default limits if no quotas are configured for a specific `client_id` or prefix.
+
+To specify a produce quota of 1GBps through the Kafka API, which means a 1GBps limit across all produce requests sent to a single Redpanda broker: 
+
+[,bash]
+----
+rpk cluster quotas alter --default client-id --add producer_byte_rate=1000000000
+----
+
+=== Bulk manage client throughput limits
 
-* The `kafka_client_group_byte_rate_quota` property applies to producer clients. It sets a maximum throughput quota of traffic sent to Redpanda from each producer in the group named by the property.
-* The `kafka_client_group_fetch_byte_rate_quota` property applies to consumer clients. It sets a maximum throughput quota of traffic fetched from Redpanda by each consumer in the group named by the property.
+To more easily manage multiple quotas, you can use the `cluster quotas describe` and `cluster quotas import` commands to do a bulk export and update.
 
-Both `kafka_client_group_byte_rate_quota` and `kafka_client_group_fetch_byte_rate_quota` have the configuration value fields:
+For example, to export all client quotas in JSON format:
 
-* `group_name`: a name for the group of clients.
-* `clients_prefix`: a client belonging to the group has this prefix in its `client_id` property.
-* `quota`: the maximum throughput rate of *each client in the group* in bytes per second.
+[,bash]
+----
+rpk cluster quotas describe --format json
+----
 
-NOTE: For `kafka_client_group_fetch_byte_rate_quota`, a group of consumer clients is not a Kafka consumer group, and the `group_name` field is not a Kafka consumer group ID. Instead, it is the set of clients fetching from a Redpanda broker that are configured by the property to be throughput limited.
+`rpk cluster quotas import` accepts the output from `rpk cluster quotas describe --format <json/yaml>`. :
 
-An example configuration of `kafka_client_group_byte_rate_quota` for two groups of producers:
+[,bash]
+----
+rpk cluster quotas import --source <path-to-output-file>
```

Comment:
I'm sorry I gave you an old example, during development we changed the flag `--source` to `from`

```suggestion
rpk cluster quotas import --from <string-or-path-to-file>
```